opentracing/src/main/java/opentracing/Tracer.java

```diff
@@ -31,7 +36,9 @@
    *                   .setTag("lucky_number", 42);
    * }</pre>
    */
-  Span startTrace(String operationName);
+  Span startSpan(String operationName, Optional<Span> parent);
+
+  <T> Injector<T> getInjector(Class<T> carrierType);
```

Comment:
let's continue this here: https://github.com/opentracing/opentracing.github.io/issues/70